fs/rpmsgfs/rpmsgfs.h

```diff
@@ -110,7 +110,9 @@ begin_packed_struct struct rpmsgfs_ioctl_s
   struct rpmsgfs_header_s header;
   int32_t                 fd;
   int32_t                 request;
-  int32_t                 arg;
+  uint64_t                arg;
+  uint32_t                arglen;
+  char                    buf[0];
```

Comment:
@xiaoxiang781216 can we switch to true C99 `buf[];` instead of GCC extension `buf[0];` if we really would like to use flexible array members in the code?

Reply:
Sure.